### PR TITLE
Hostname change playbook

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,3 +57,17 @@ This project can be used to quickly setup Satellite 6.1 or 6.2 install with the
...
+2. Update the required sections in `roles/change_hostname/vars/main.yml`.
+3. Update the `inventory` file under the root of this project to add your Satellite VM's ip address.
+4. Run the ansible playbook:
+    ```console
+      # ansible-playbook -i inventory deploy-hostname-playbook.yml
+    ```
+   or if wanting to execute the playbook on localhost, specify the local IP address in the inventory file and run:
+    ```console
+      # ansible-playbook -i inventory -c local deploy-hostname-playbook.yml
+    ```
```
Comment (sthirugn) [inline README.md]: Lines 70-73 are redundant.  You can very well specify the localhost ip address in inventory and run Line 68 without issues.

Comment (johnpmitsch) [inline README.md]: I can't run it this way and need the `-c local`

```
[root@sat-snap-rhel7 satellite-clone]# ansible-playbook -i inventory deploy-hostname-playbook.yml 

PLAY [satellite6] **************************************************************

TASK [setup] *******************************************************************
fatal: [192.168.121.95]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: Permission denied (publickey,gssapi-keyex,gssapi-with-mic,password).\r\n", "unreachable": true}
        to retry, use: --limit @/root/satellite-clone/deploy-hostname-playbook.retry

PLAY RECAP *********************************************************************
192.168.121.95             : ok=0    changed=0    unreachable=1    failed=0   

 ------------------------------------------------------------------------ 0.23s

Playbook finished: Tue Nov 29 17:33:48 2016, 1 total tasks.  0:00:00 elapsed. 
```

Comment (johnpmitsch) [inline README.md]: Are you able to run the playbook successfully on the local machine?

File: inventory [hunk 2]
```diff
@@ -1,2 +1,2 @@
 [satellite6]
-192.168.1.100
+192.168.121.95
```
Comment (sthirugn) [inline inventory]: this change is not required

**Comment by sthirugn** [comment]
ACK

**Pull request merged**